packages/vue/src/components/confirm-sign-up.vue

```diff
@@ -143,13 +146,15 @@ export default defineComponent({
 
     const submit = (e: Event): void => {
       const formData = new FormData(<HTMLFormElement>e.target);
+      const { user, authAttributes } = actorContext;
+      const username = user?.username ?? authAttributes?.username;
       send({
         type: 'SUBMIT',
         //@ts-ignore
         data: {
           //@ts-ignore
           ...Object.fromEntries(formData),
-          username: context?.formValues[primaryAlias],
+          username: username,
```

Comment:
You fixed it for submit but you missed updating it for `onLostCodeClicked`. That's still trying to pull from formValues where it should be sending over the `username` like on the submit method.

Reply:
Thanks for the catch -- https://github.com/aws-amplify/amplify-ui/pull/180/commits/417d0775068b9d66ccc2683abc9f9437fa56e6e6 to remove primaryAlias as well as cover onLostCodeClicked